README.md

```diff
@@ -44,6 +55,16 @@ combineReducers({
 Now you won't be able to increment the counter anymore, because the
 `INCREMENT_COUNTER` action is ignored.
 
+Alternatively, you can ignore actions via a predicate function:
+
+```js
+combineReducers({
+  counter: ignoreActions(counter, (action) => !action.entity === 'COUNTER')
```

Comment:
maybe use the same example as above here and filter out the type `INCREMENT_COUNTER`? or do you think it's better to use a different example?